transformer_engine/common/transformer_engine.cpp

```diff
@@ -92,8 +92,8 @@ void CheckScaleTensorShape(const Tensor &t) {
         expected_y =
             DIVUP(DIVUP(t.flat_last_dim(), static_cast<size_t>(32)), alignment) * alignment;
         const auto &expected = std::vector<size_t>{expected_x, expected_y};
-        NVTE_CHECK(t.scale_inv.shape == expected, "Tensor has invalid scale_inv shape (expected ",
-                   expected, ", got ", t.scale_inv.shape, ")");
+        // NVTE_CHECK(t.scale_inv.shape == expected, "Tensor has invalid scale_inv shape (expected ",
+        //            expected, ", got ", t.scale_inv.shape, ")");
```

Comment:
Don't we still expect the shape to be aligned to `(128, 4)`?
```suggestion
        NVTE_CHECK(t.scale_inv.shape == expected, "Tensor has invalid scale_inv shape (expected ",
                   expected, ", got ", t.scale_inv.shape, ")");
```